src/runtime/storage/firebase/firebase-storage.ts

```diff
@@ -1438,7 +1438,7 @@ class FirebaseBigCollection extends FirebaseStorageProvider implements BigCollec
   }
 
   /** Calls close() on and discards the cursor identified by cursorId. */
-  cursorClose(cursorId: number) {
+  async cursorClose(cursorId: number): Promise<void> {
```

Comment:
This doesn't need to be async.

Reply:
I'm pretty sure that `cursorClose` needs to be async in `StorageProxy` to cross the message port so I thought I'd align the APIs.

```
  cursorClose(cursorId): Promise<void> {
    this.port.StreamCursorClose(this, cursorId);
    return Promise.resolve();
  }
```

Reply:
Every API channel call is message-based, but cursorClose doesn't need a response, so it can be fire-and-forget - there's no need to force a wait point in the calling code.

Reply:
Thanks, moved back to non-async.